content/docs/other-guides/authentication.md

```diff
@@ -0,0 +1,514 @@
++++
+title = "Kubeflow Authentication with Istio and Dex"
+description = "Instructions for installing Kubeflow with authentication support"
+weight = 4
++++
+
+This installation of Kubeflow is maintained by [Arrikto](https://www.arrikto.com/), is geared towards existing Kubernetes clusters and does not depend on any cloud-specific feature.
```

Comment:
Maybe drop by "maintained by [Arrikto]"
These instructions are clearly about Istio and Dex. So I'm hoping there's a general consensus around what this should be doing.

Reply:
done

Reply:
done